### PR TITLE
Cypress test

### PR DESCRIPTION
**Pull request by Jin1L**

### CODE REVIEW

File: cypress/e2e/04_env_variable_modal.cy.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    cy.get('[data-testid="EnvVarKeyInput"]').click().type("CypressTest");
+
+    cy.percySnapshot("dark flow page with key input");
+
+    cy.get('[data-testid="EnvVarValueInput"]').click().type("CypressTest");
+
+    cy.percySnapshot("dark flow page with value input");
+
+    cy.get('[data-testid="envModalAddBtn"]').click();
```
Comment (JeffDotPng) [inline cypress/e2e/04_env_variable_modal.cy.ts:31]: Can you be consistent with the `data-testid` naming conventions? There's a mix of kebab-case, PascalCase, and camelCase.. (kebab-case is usually what's used for html ids so you should stick to that)

File: cypress/e2e/04_env_variable_modal.cy.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+
+import { text } from "stream/consumers";
+
+describe("Verify Env Variable Modal", () => {
+  // The interactions use typical Cypress calls,
+  // but the verifications use one-line snapshot calls with Applitools Eyes.
+  // If the page ever changes, then Applitools will detect the changes and highlight them in the Eyes Test Manager.
+  // Traditional assertions that scrape the page for text values are not needed here.
```
Comment (JeffDotPng) [inline cypress/e2e/04_env_variable_modal.cy.ts:9]: You can remove these comments

File: cypress/e2e/04_env_variable_modal.cy.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  it("env variable modal test", () => {
+    cy.visit("/").wait(1000);
+
+    // Clear canvas
```
Comment (JeffDotPng) [inline cypress/e2e/04_env_variable_modal.cy.ts:14]: This too

File: cypress/e2e/10_nodes_and_params.cy.ts [hunk 4]
```diff
@@ -148,20 +148,20 @@ describe("Verify nodes and its parameters", () => {
     cy.get('[data-testid="darkmode-toggle"]').click();
 
     // creating loop node
-    cy.get('[data-testid="add-node-button"]').click();
```
Comment (JeffDotPng) [inline cypress/e2e/10_nodes_and_params.cy.ts:151]: What happened to the loop test?

File: cypress/e2e/16_command_menu.cy.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress" />
+
+//** Tests app gallery feature.*/
+
+describe("Command test", () => {
+  it("Command test", () => {
+    cy.visit("/").wait(1000);
+
+    cy.get('[data-testid="react-flow"]').trigger("keydown", {
```
Comment (JeffDotPng) [inline cypress/e2e/16_command_menu.cy.ts:9]: Maybe add a check after this to make sure that something actually happened?

File: flojoy [hunk 6]
```diff
@@ -170,7 +170,8 @@ createFlojoyDirectoryWithYmlFile()
       mkdir "$FOLDER" && touch "$FILE" && echo "PATH: $CWD" > "$FILE"
       feedback $? "Created new $FOLDER directory with flojoy.yaml file." 'Failed to create file in the home directory, check the permission or sign in as root user'
    fi
-   CREDENTIALS_FILE=$FOLDER/credentials
+   CREDENTIALS_FILE=$FOLDER/credentials.txt
+   touch $CREDENTIALS_FILE
```
Comment (JeffDotPng) [inline flojoy:174]: Variables in bash should always be wrapped in quotes (e.g `touch "$CREDENTIALS_FILE"`)

File: flojoy [hunk 7]
```diff
@@ -170,7 +170,8 @@ createFlojoyDirectoryWithYmlFile()
...
    if ! test -f "$CREDENTIALS_FILE"; then
```
Comment (JeffDotPng) [inline flojoy:175]: You can remove this check since you're creating it if it doesn't exist now

**Review by JeffDotPng — CHANGES_REQUESTED** [review]

File: cypress/e2e/16_command_menu.cy.ts (hunk 5 above)
Comment (Jin1L) [reply cypress/e2e/16_command_menu.cy.ts:9]: I was planning to check it visually using Percy. Do you think it's a bad idea?

Comment (JeffDotPng) [reply cypress/e2e/16_command_menu.cy.ts:9]: Yeah that's good

**Review by JeffDotPng — APPROVED** [review]
looks good

**Pull request merged**